inc/languages/english/tickets.lang.php

```diff
@@ -1,24 +1,24 @@
 <?php
-$l['toplinks_tickets'] = "Ticketsystem";
+$l['toplinks_tickets'] = "Ticket system";
 $l['ticket'] = "Ticket";
 $l['tickets'] = "Tickets";
-$l['tickets_deactivated'] = "Ticketsystem is deactivated";
+$l['tickets_deactivated'] = "The Ticket system is deactivated";
 $l['no_id'] = "No ID send";
 $l['wrong_id'] = "Wrong ID send";
 $l['tickets_admin'] = "Admin Section";
 
 $l['ticket_title'] = "Title";
 $l['ticket_created_at'] = "Created";
-$l['ticket_answers'] = "Replys";
+$l['ticket_answers'] = "Add Reply";
```

Comment:
IIRC this language string is also used somewhere else (in the listing?) so it shouldn't be changed, instead I need to add a new one. However I'm aware of it, just never fixed it.